### DISCUSSION TITLE
error at (5, 12) class dialogueresource hides a global script class

### DISCUSSION THREAD

**Discussion by dewdoughnut** [nathanhoad/godot_dialogue_manager#477] [Q&A]
i dont know whats wrong. i simply tried running godot after updating and i keep getting this error and cant use the dialogue manager. is it a godot thing, a "its out of date thing or something else"

![Screenshot 2024-02-12 180445](https://github.com/nathanhoad/godot_dialogue_manager/assets/56401177/4f7cbb3f-1f38-4c1c-891c-bad4edc406fd)

**Comment by dewdoughnut** [comment]
I worked out the problem... turns out I missed something small and silly, the dialogue resource didnt reference a dialogue script and the start point was named wrong.

**Reply by TheWalruzz** [reply]
Can you give me more information on how you fixed that? I'm bashing my head against it and DialogueResource has correct script attached (dialogue_resource.gd) and the dialogue file is the example one with Nathan in it.

**Reply by TheWalruzz** [reply]
Nevermind, something strange happened. I've been setting up the GDM in my project and at first POT generation was not set up, so it has thrown an error after I created a dialogue file. This somehow corrupted it and I was unable to do anything. After I removed the file and created a new one it started working again.

**Comment by Lykdraft** [comment]
Yeah, well... same problem at line 5:

> class_name DialogueResource extends Resource

 Still no solution months later. Weird...
